internal/ethapi/api.go

```diff
@@ -32,6 +32,21 @@ import (
 	"github.com/ledgerwatch/log/v3"
 )
 
+// Covalent change: Empty Backend interface used for PublicBlockChainAPI below
```

Comment:
Please do not bring this back, all RPC comms need to go via RPC Daemon. We are also trying to gradually retire Backend approach. This package is currently only used to contain some types and functions that are used in RPC Daemon, and needs some cleanup

Reply:
What should I use an alternative programmatically?

Reply:
The code for extra RPC currently goes into `cmd/rpcdaemon/commands`, as I mentioned in one of the previous comment